demo/demo.md

```diff
@@ -191,74 +178,100 @@ The following example is static to illustrate a error state returned with page l
-  <auro-input id="validation1" required validateOnInput pattern="[a-zA-Z-.']+( +[a-zA-Z-.']+)+" setCustomValidity="Full name requires two or more names with at least one space.">
+  <auro-input id="error10" required minlength="3" setCustomValidity="Sorry, please enter your first and last name (one space required).">
     <span slot="label">Full Name</span>
-    <span slot="helptext">Please enter your full name as it appears on the card.</span>
+    <span slot="helptext">Please enter your full name.</span>
   </auro-input>
-  <auro-input id="validation1" bordered validateOnInput required pattern="[a-zA-Z-.']+( +[a-zA-Z-.']+)+" setCustomValidity="Full name requires two or more names with at least one space.">
+  <auro-input id="error20" bordered required minlength="3" setCustomValidity="Sorry, please enter your first and last name (one space required).">
     <span slot="label">Full Name</span>
-    <span slot="helptext">Please enter your full name as it appears on the card.</span>
+    <span slot="helptext">Please enter your full name.</span>
   </auro-input>
 </div>
 
 <auro-accordion lowProfile justifyRight>
   <span slot="trigger">See code</span>
 
   ```html
-  <auro-input id="validation1" required validateOnInput pattern="[a-zA-Z-.']+( +[a-zA-Z-.']+)+" setCustomValidity="Full name requires two or more names with at least one space.">
+  <auro-input id="error10" required setCustomValidity="Sorry, please enter your first and last name (one space required).">
     <span slot="label">Full Name</span>
-    <span slot="helptext">Please enter your full name as it appears on the card.</span>
+    <span slot="helptext">Please enter your full name.</span>
   </auro-input>
-
-  <auro-input id="validation1" bordered validateOnInput required pattern="[a-zA-Z-.']+( +[a-zA-Z-.']+)+" setCustomValidity="Full name requires two or more names with at least one space.">
+  <auro-input id="error20" bordered required setCustomValidity="Sorry, please enter your first and last name (one space required).">
     <span slot="label">Full Name</span>
-    <span slot="helptext">Please enter your full name as it appears on the card.</span>
+    <span slot="helptext">Please enter your full name.</span>
+  </auro-input>
+  ```
+</auro-accordion>
+
+### Error Attribute
+
+Use the `error` attribute to apply a persistent custom error that supersedes the HTML5 validation logic.
+
+<auro-button id="setCustomErrorBtn">Set Custom Error</auro-button>
+<auro-button id="setCustomErrorClearBtn">Clear Custom Error</auro-button>
+
+<div class="exampleWrapper">
+  <auro-input id="setCustomErrorExample" error="Initial error attribute value">
+    <span slot="label">Name</span>
+  </auro-input>
+</div>
+
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+  ```html
+  <auro-input id="error2" required bordered error="Please enter your full name.">
+    <span slot="label">Name</span>
+    <span slot="helptext">Please enter your full name.</span>
```

Comment:
This update to the demo is a little confusing. There is no representation of the `error` attribute in the demo code. 

There is an error with the new `button` features. See in this screen grab, the error is not actually cleared. The `alert` icon simply turns `gray`. 

![errorAttr](https://user-images.githubusercontent.com/181089/191107395-65ec4e60-db14-4d97-9fdc-a69593b716ea.gif)

Reply:
> This update to the demo is a little confusing. There is no representation of the `error` attribute in the demo code.
> 
> There is an error with the new `button` features. See in this screen grab, the error is not actually cleared. The `alert` icon simply turns `gray`.

@blackfalcon I don't understand. The error attribute is in the example code and the rendered example. The validity error and the custom message are also rendered correctly when loading the component. As well as using the new button to show changing the error message via JS.

It seems represented to me. Am I misunderstanding what you mean?

Good catch on the icon when clearing the error. [This comment](https://github.com/AlaskaAirlines/auro-input/pull/180/commits/b48a708e22bcf1715ea8f1aaa12f160d95a7b9fc) fixes that.

Reply:
What is in the string is not in the example? The `error` in the code says, "Please enter your full name."

In the UI, there is `Custom error message` and `initial error attribute value` whatever that means? 

Also, when you `Clear Custom Error`, the `help text` does not come back.

Reply:
@blackfalcon Ah I see, that is a copy/paste error due to auro-input not using the new apiExamples doc system. I fixed that part. 

Good catch on the error reset not just showing the slotted helpText when the error is cleared. Looking into that now.

Reply:
@blackfalcon Fixed - I was showing the helpText when validity was `undefined` or `valid` but not when it was `null`.

Nice catch.